### ISSUE TITLE
Questions about Training Details and Command for MAR-L on ImageNet 512x512

### ISSUE THREAD

**Issue by Tom-zgt** [LTH14/mar#85]
Hello, @LTH14. I am following your great work. 

I noticed that you provided results for MAR-L on ImageNet 512×512, and I would like to ask for more details regarding the training setup.

Could you kindly share the specific training hyperparameters, including:

Batch learning rate (**blr**)
Batch size (**batch_size**)
Warm-up epochs (**warmup_epochs**)
Diffusion batch multiplier (**diffusion_batch_mul**)
Configuration used during testing (**cfg**)
Additionally, it would be extremely helpful if you could provide the exact command used for training, similar to the following example:
torchrun --nproc_per_node=8 --nnodes=4 --node_rank=${NODE_RANK} --master_addr=${MASTER_ADDR} --master_port=${MASTER_PORT} \
main_mar.py \
--img_size 256 --vae_path pretrained_models/vae/kl16.ckpt --vae_embed_dim 16 --vae_stride 16 --patch_size 1 \
--model mar_large --diffloss_d 3 --diffloss_w 1024 \
--epochs 400 --warmup_epochs 100 --batch_size 64 --blr 1.0e-4 --diffusion_batch_mul 4 \
--output_dir ${OUTPUT_DIR} --resume ${OUTPUT_DIR} \
--data_path ${IMAGENET_PATH}

**I have also sent an email** inquiring about more detailed information. If you could kindly respond, I would be truly grateful.

Looking forward to your response!

**Comment by LTH14** [comment]
Thanks for your interest! The current codebase does not directly support IN512 training. You need to change the VAE to the one provided by SD (which is also used by DiT).
